src/App.css

```diff
@@ -20,3 +20,47 @@
 .App-link {
   color: #09d3ac;
 }
+
+/* html, body, div, span, applet, object, iframe,
```

Comment:
You can delete all the default css  created by the create-react-app with no worries